packages/database/src/migrations/20210112015125-AddHeatmapContactsConfirmedSuspectedCasesSCovidSamoa-modifies-data.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+import { insertObject, generateId, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+async function deleteMapOverlay(db, mapOverlayId) {
+  return db.runSql(`
+    DELETE FROM
+      "mapOverlay"
+    WHERE
+      "id" = '${mapOverlayId}';
+  `);
+}
+
+async function deleteMapOverlayInMapOverlayGroupRelation(db, childId) {
+  return db.runSql(`
+    DELETE FROM
+      "map_overlay_group_relation"
+    WHERE
+      "child_id" = '${childId}';
+  `);
+}
+
+const mapOverlay = {
+  id: 'COVID_WS_Address_Contacts_Of_Suspected_And_Confirmed_Cases_WS',
+  name: 'Home village of contacts of suspected/confirmed cases',
+  userGroup: 'Donor',
+  dataElementCode: 'value',
+  measureBuilderConfig: {
+    dataValues: {
+      WS_CD_CT_Contact_12: '*',
```

Comment:
That is not required since we're getting all responses in the selected village anyway. This happens via the `entityAggregation` field below, which groups cases by their village.

But it's good that you included it, because it made me realise an issue in the survey! https://github.com/beyondessential/tupaia-backlog/issues/1623#issuecomment-759249393 😄 

Better to remove this since this question will be removed in the new survey

```suggestion
```